aepsych/config.py

```diff
@@ -104,6 +105,27 @@ def _get(
                 **kwargs,
             )
 
+    # Convert config into a dictionary (eliminate duplicates from defaulted 'common' section.)
+    def to_dict(config):
```

Comment:
It looks like this was copied from `__str__`, which is fine, but maybe this implies that we should have a helper function that both methods use so that we're not duplicating code.